desktop/dialogs/AddPodcastsFromOPMLDialog.qml

```diff
@@ -0,0 +1,36 @@
+import QtQuick 2.0
+import QtQuick.Controls 1.0
+import QtQuick.Layouts 1.0
+import QtQuick.Dialogs 1.2
+
+import '../common'
+import '../common/util.js' as Util
+
+Dialog {
```

Comment:
Since this is mostly the same as the "add podcast from URL" dialog, maybe parameterize the former one and use the parameterized version in both uses.